### PR TITLE
Login dev issues 2

### PR DESCRIPTION
**Pull request by asif986**
1. when logout the  user into arduino then Esim automatic logged . this issuess fixed.
2. Hiding the token querparams in production mode.
3. Not neccessary to add cors and application json key in http Header.

### CODE REVIEW

File: eda-frontend/src/redux/actions/authActions.js [hunk 1]
```diff
@@ -154,8 +156,8 @@ export const login = (username, password, toUrl) => {
             window.open('', '_self')
             window.close()
           } else {
-            window.open(toUrl, '_self')
             localStorage.setItem('ard_redurl', '')
+            window.location.href = toUrl+"?token=" + localStorage.getItem('esim_token');
```
Comment (kumanik5661) [inline eda-frontend/src/redux/actions/authActions.js:160]: Maybe a security concern? 
Normally I wouldn't want to expose the token like this via the URL

File: eda-frontend/src/redux/actions/authActions.js [hunk 2]
```diff
@@ -125,7 +126,8 @@ export const loadMinUser = () => (dispatch) => {
         type: actions.LOGIN_FAILED,
         payload: {
           data: {}
-        }
+        },
```
Comment (kumanik5661) [inline eda-frontend/src/redux/actions/authActions.js:129]: Extra comma might interfere(?)

**Review by kumanik5661 — APPROVED** [review]
Other things seem fine

**Review by kumanik5661 — CHANGES_REQUESTED** [review]
Other things seem fine

File: eda-frontend/src/redux/actions/authActions.js (hunk 2 above)
Comment (asif986) [reply eda-frontend/src/redux/actions/authActions.js:129]: Ok . I can remove.

**Pull request merged**